src/items/itemsTable.component.tsx

```diff
@@ -219,23 +239,58 @@ export function ItemsTable(props: ItemTableProps) {
         Header: TableHeaderOverflowTip,
         accessorFn: (row) => (row.item.is_defective === true ? 'Yes' : 'No'),
         id: 'item.is_defective',
+        filterVariant: COLUMN_FILTER_VARIANTS['boolean'],
+        enableColumnFilterModes: false,
         size: 200,
-        filterVariant: 'autocomplete',
+        Cell: ({ row }) => (row.original.item.is_defective ? 'Yes' : 'No'),
       },
       {
         header: 'Usage Status',
         Header: TableHeaderOverflowTip,
         accessorFn: (row) => row.item.usage_status,
         id: 'item.usage_status',
-        size: 200,
-        filterVariant: 'autocomplete',
+        filterVariant: 'multi-select',
+        filterFn: 'arrIncludesSome',
+        columnFilterModeOptions: ['arrIncludesSome', 'arrIncludesNone'],
+        renderColumnFilterModeMenuItems: ({ onSelectFilterMode }) => [
+          <MenuItem
+            key="arrIncludesSome"
+            onClick={() => onSelectFilterMode('arrIncludesSome')}
+          >
+            Includes
```

Comment:
Can you add units test any additional logic you added that wasn't built into MRT.  i.e "customFilterFunctions"

Reply:
No, high code coverage alone doesn’t guarantee thorough testing, as it only shows that lines were executed, not that all cases were covered.